Rakefile

```diff
@@ -1133,18 +1127,19 @@ namespace :release do
   end
 
   task :homebrew_cask do
+    version = install_cocoapods_version
+
     cask_fork = JSON.load(REST.post("https://api.github.com/repos/caskroom/homebrew-cask/forks?access_token=#{github_access_token}",
                                     {}.to_json,
                                     github_headers).body)["full_name"]
-
     branch = "cocoapods-#{version}"
     message = "Upgrade CocoaPods to v#{version}"
+    puts cask_fork
 
-    rm_rf "homebrew_cask"
-    sh "git clone https://github.com/caskroom/homebrew-cask.git --depth=1"
+    sh "git clone https://github.com/caskroom/homebrew-cask.git homebrew_cask" unless Dir.exists? "homebrew_cask"
```

Comment:
shame about the depth thing, but I couldn't push to my fork when it only had a depth of 1